src/App.jsx

```diff
@@ -1,20 +1,25 @@
 import React from 'react';
+import { Provider } from 'react-redux';
 import { BrowserRouter } from 'react-router-dom';
 import Routes from './routes/Routes';
+import store from './store/store';
+import Recipe from './pages/Recipe/Recipe';
 
-const envVar = process.env.ENV_VAR;
 /**
  * @description - App component
  * @param {object} props
- * @returns {JSX} - Returns all Components
+ * @returns {JSX} - Retuerns all Component
  */
 const App = () => (
-  <BrowserRouter>
-    <div>
-      <h1>Welcome to Naija Chop Chop {envVar}</h1>
-      <Routes />
-    </div>
-  </BrowserRouter>
+  <Provider store={store}>
+    <BrowserRouter>
+      <div>
+        <h1>Welcome to Naija Chop Chop</h1>
+        <Recipe />
```

Comment:
I'm a little confused here. Is the aim to render a list of recipes _and then_ the app routes?

Reply:
yes. All is good to go now.